client/state/form/shipping-label/actions.js

```diff
@@ -1,9 +1,13 @@
+import * as FormActions from 'state/form/actions';
+
 export const OPEN_PRINTING_FLOW = 'OPEN_PRINTING_FLOW';
 export const EXIT_PRINTING_FLOW = 'EXIT_PRINTING_FLOW';
 
-export const openPrintingFlow = () => ( {
-	type: OPEN_PRINTING_FLOW,
-} );
+export const openPrintingFlow = () => ( dispatch ) => {
+	dispatch( { type: OPEN_PRINTING_FLOW } );
+	dispatch( FormActions.setFormProperty( 'pristine', false ) );
+	dispatch( FormActions.nextStep() );
+};
```

Comment:
I've tried, but... https://github.com/tshelburne/redux-batched-actions/issues/3
We can't use (for now) batched actions with thunks inside. It would be a good way to contribute back if we feel that our life is empty without this feature :)